pytorch_lightning/logging/sacred.py

```diff
@@ -0,0 +1,65 @@
+from logging import getLogger
+from time import time
+
+try:
+    import sacred
+except ImportError:
+    raise ImportError('Missing sacred package.')
+
+from .base import LightningLoggerBase, rank_zero_only
```

Comment:
pls use absolute imports

Reply:
I copied it from here: https://github.com/williamFalcon/pytorch-lightning/blob/master/pytorch_lightning/logging/test_tube.py#L37

Reply:
sure, but I think that we commented on the importing too :]